svd/patches/_interrupts.yaml

```diff
@@ -0,0 +1,490 @@
+# Draft attempt at adding interrupt mappings to peripherals.
```

Comment:
This whole project is WIP 😂 , may as well remove this As I'm sure we'll make changes in the future